docs/data/joy/components/grid/AutoGrid.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { styled } from '@mui/joy/styles';
+import Box from '@mui/joy/Box';
+import Sheet from '@mui/joy/Sheet';
+import Grid from '@mui/joy/Grid';
+
+const Item = styled(Sheet)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: 'center',
+  color: theme.vars.palette.neutral[500],
+}));
+
+export default function AutoGrid() {
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <Grid container spacing={3}>
```

Comment:
```suggestion
      <Grid container spacing={3} sx={{ flexGrow: 1 }}>
```

I guess the Box could be removed to reduce the lines of code.